homeassistant/components/broadlink/climate.py

```diff
@@ -61,6 +71,7 @@ async def async_set_temperature(self, **kwargs: Any) -> None:
     @callback
     def _update_state(self, data: dict[str, Any]) -> None:
         """Update data."""
+        self.sensor_mode = SensorMode(data.get("sensor"))
```

Comment:
If `sensor` is not part of the data an error will be raised as `SensorMode(None)` will raise the `ValueError`
Are we sure that `sensor` is always included in the data?
If yes, than we should change it to `data["sensor"]`
If not we should only update the value if sensor is included